zerver/tests/test_openapi.py

```diff
@@ -819,6 +819,7 @@ def test_generate_and_render_curl_with_array_example(self) -> None:
             "curl -sSX GET -G http://localhost:9991/api/v1/messages \\",
             "    -u BOT_EMAIL_ADDRESS:BOT_API_KEY \\",
             "    --data-urlencode anchor=43 \\",
+            "    --data-urlencode anchor_date=2023-05-18T00:00:00Z \\",
```

Comment:
We probably don't want to add these in this file, since the whole point is to have a real example input, and this parameter should not be used with that value of anchor.

Reply:
Ah, I missed

Reply:
I'm not very sure how to pass the test without changing the example anchor value to `date`.

Reply:
If we don't want `anchor_date` documented here, we can try setting `intentionally_undocumented` of `REQ` might be helpful.

Reply:
Oh, right then.I guess we don't want to do that. I think we want to pass `exclude=["anchor_date"]`. May be look at other places in the file where we skip few fields.

Reply:
Yeah, that `exclude` option sounds correct.